pipeline/common/downloads.py

```diff
@@ -175,7 +184,7 @@ class DownloadChunkStreamer(io.IOBase):
              gzip.GzipFile(fileobj=f)
     """
 
-    def __init__(self, url: str, total_retries=3, timeout_sec=10.0, wait_before_retry_sec=60.0):
+    def __init__(self, url: str, total_retries=3, timeout_sec=10.0, wait_before_retry_sec=1.0):
```

Comment:
Is this an intentional to change? I know you had wanted to limit being rate limited by some of these download services.

Reply:
No, I was testing something locally, good catch